src/ext-test/java/org/opentripplanner/ext/fares/impl/GtfsFaresV2ServiceTest.java

```diff
@@ -121,14 +121,23 @@ class GtfsFaresV2ServiceTest implements PlanTestConstants {
 
   GtfsFaresV2Service service = new GtfsFaresV2Service(
     List.of(
-      new FareLegRule(LEG_GROUP1, null, null, null, single),
-      new FareLegRule(LEG_GROUP1, null, null, OUTER_ZONE, singleToOuter),
-      new FareLegRule(LEG_GROUP1, null, OUTER_ZONE, null, singleFromOuter),
-      new FareLegRule(LEG_GROUP1, null, null, null, dayPass),
-      new FareLegRule(LEG_GROUP1, expressNetwork, null, null, expressPass),
-      new FareLegRule(LEG_GROUP1, localNetwork, null, null, localPass),
-      new FareLegRule(LEG_GROUP1, null, INNER_ZONE, OUTER_ZONE, innerToOuterZoneSingle),
-      new FareLegRule("another-leg-group", null, null, null, monthlyPass)
+      new FareLegRule(LEG_GROUP1, null, null, null, null, null, null, single),
```

Comment:
Can you add constructors to `FareLegRule` so you don't have to repeat `null` quite so often?